internal/k8s/rbac.go

```diff
@@ -0,0 +1,26 @@
+// Copyright Project Contour Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8s
+
```

Comment:
unrelated to this change, but we also have https://github.com/projectcontour/contour/blob/7fa2a535d1112841fa5a21338ba76a3eaa1cac78/cmd/contour/serve.go#L64-L65

which might be good to consolidate in this file too?

Reply:
might be good as a TODO issue after this is merged

Reply:
oh yeah definitely makes sense to move that in now, thanks for pointing it out. I'll just update it now to avoid another PR.